src/features/genres/genresSlice.js

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const genresSlice = createSlice(
+    {
+        name: "genres",
+        initialState: {
+            genres: [],
+            loading: false,
```

Comment:
Tak się zastanawiam czy ten loading jest tutaj potrzebny?

Reply:
Chyba nie, bo i tak nigdzie z takiego stanu nie będziemy korzystać

Reply:
To prawda, aczkolwiek moim zamysłem tutaj było mimo wszystko utrzymanie konwencji, którą stosowaliśmy w poprzednich slice'ach oraz jeśli gdzieś byśmy myśleli o późniejszej rozbudowie aplikacji, to taki stan gdzieś tam może się przydać, więc ja bym go w sumie raczej nie usuwał :)